website/server/models/subscriptionPlan.js

```diff
@@ -29,6 +32,7 @@ export let schema = new mongoose.Schema({
 });
 
 schema.plugin(baseModel, {
+  private: 'additionalData',
```

Comment:
this should be an array of strings not a single string, also I think this isn't needed because the user.purchased object is already blacklisted at the user model level